components/mail/thunderbird/Makefile

```diff
@@ -108,7 +109,7 @@ COMPONENT_POST_INSTALL_ACTION += \
 	unzip -u -d $(LIGHTNING_DIR) $(USERLAND_ARCHIVES)$(COMPONENT_ARCHIVE_1); \
 	for file in $(PROTO_DIR)/usr/lib/thunderbird-$(COMPONENT_VERSION)/*.so \
           $(PROTO_DIR)/usr/lib/thunderbird-$(COMPONENT_VERSION)/plugin-container; do \
-	  /usr/bin/elfedit -e 'dyn:value -s  RPATH "/usr/lib:/usr/lib/thunderbird"' $$file ;\
+	  /usr/bin/elfedit -e 'dyn:value -s  RPATH "/usr/lib:/usr/lib/thunderbird:$$ORIGIN:$$ORIGIN/.."' $$file ;\
```

Comment:
What are you trying to do? /usr/lib/thunderbird == $ORIGIN for these libraries...